tests/pmdarima/test_pmdarima_model_export.py

```diff
@@ -114,54 +114,51 @@ def test_pmdarima_autoarima_pyfunc_save_and_load(auto_arima_model, model_path):
     np.testing.assert_array_equal(yhat_high, pyfunc_predict["yhat_upper"])
 
 
-def test_pmdarima_signature_and_examples_saved_correctly(auto_arima_model, test_data):
+@pytest.mark.parametrize("_signature", [True, False])
+@pytest.mark.parametrize("_example", [True, False])
+def test_pmdarima_signature_and_examples_saved_correctly(
+    auto_arima_model, test_data, model_path, _signature, _example
+):
 
     # NB: Signature inference will only work on the first element of the tuple return
     prediction = auto_arima_model.predict(n_periods=20, return_conf_int=True, alpha=0.05)
-    signature_ = infer_signature(test_data, prediction[0])
-    example_ = test_data[0:5].copy(deep=False)
-    for signature in (None, signature_):
-        for example in (None, example_):
-            with TempDir() as tmp:
-                path = tmp.path("model")
-                mlflow.pmdarima.save_model(
-                    auto_arima_model, path=path, signature=signature, input_example=example
-                )
-                mlflow_model = Model.load(path)
-                assert signature == mlflow_model.signature
-                if example is None:
-                    assert mlflow_model.saved_input_example_info is None
-                else:
-                    r_example = _read_example(mlflow_model, path).copy(deep=False)
-                    np.testing.assert_array_equal(r_example, example)
+    signature = infer_signature(test_data, prediction[0]) if _signature else None
+    example = test_data[0:5].copy(deep=False) if _example else None
+    mlflow.pmdarima.save_model(
+        auto_arima_model, path=model_path, signature=signature, input_example=example
+    )
+    mlflow_model = Model.load(model_path)
+    assert signature == mlflow_model.signature
+    if example is None:
+        assert mlflow_model.saved_input_example_info is None
+    else:
+        r_example = _read_example(mlflow_model, model_path).copy(deep=False)
+        np.testing.assert_array_equal(r_example, example)
 
 
+@pytest.mark.parametrize("_signature", [True, False])
+@pytest.mark.parametrize("_example", [True, False])
```

Comment:
```suggestion
@pytest.mark.parametrize("use_signature", [True, False])
@pytest.mark.parametrize("use_example", [True, False])
```

Can we rename the parameters?

Reply:
great idea! Changed!